services/powchain/src/main/java/tech/pegasys/teku/services/powchain/execution/client/schema/ExecutePayloadResponse.java

```diff
@@ -36,17 +41,17 @@ public boolean equals(Object o) {
     if (o == null || getClass() != o.getClass()) {
       return false;
     }
-    NewBlockResponse that = (NewBlockResponse) o;
-    return Objects.equals(valid, that.valid);
+    ExecutePayloadResponse that = (ExecutePayloadResponse) o;
+    return Objects.equals(status, that.status);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(valid);
+    return Objects.hash(status);
   }
 
   @Override
   public String toString() {
-    return "NewBlockResponse{" + "valid=" + valid + '}';
+    return "NewBlockResponse{" + "valid=" + status.name() + '}';
```

Comment:
Makes sense to fix the name?

Reply:
Now I refreshed the how enum works, you're right :)